opendevin/server/agent/agent.py

```diff
@@ -93,7 +93,7 @@ async def dispatch(self, action: str | None, data: dict):
             case ActionType.INIT:
                 await self.create_controller(data)
             case ActionType.START:
-                await self.start_task(data)
+                await self.start_task_or_chat(data)
```

Comment:
We shouldn't be sending a `START` if it's a chat interruption--I think this should be a separate `case`

Reply:
There's currently a status indicator on the frontend, which shows agent status--so it's there somewhere!